packages/frontend/src/hooks/thirdPartyServices/useSentry.ts

```diff
@@ -17,7 +20,15 @@ const useSentry = (sentryConfig: HealthState['sentry'] | undefined) => {
             });
             setIsSentryLoaded(true);
         }
-    }, [isSentryLoaded, setIsSentryLoaded, sentryConfig]);
+        if (user) {
+            Sentry.setUser({
+                id: user.userUuid,
+                email: user.email,
+                username: user.email,
+                segment: user.organizationUuid,
+            });
+        }
+    }, [isSentryLoaded, setIsSentryLoaded, sentryConfig, user]);
```

Comment:
We should move this to a separate hook.

Reply:
I was following the useCohere hook pattern, do you mean a separate useEffect or a totally new hook? I'm not sure what happens if we try to `setUser` and sentry isn't yet initialised